commands/utils/params.go

```diff
@@ -97,10 +97,11 @@ func GetParamsAndClient() (configAggregator FrogbotConfigAggregator, server *cor
 		return nil, nil, nil, err
 	}
 
-	configData, err := getFrogbotConfig(&gitParams, client)
+	configData, err := getFrogbotConfig(client)
 	// If the error is due to missing configuration, try to generate an environment variable-based config aggregator.
 	if _, missingConfigErr := err.(*ErrMissingConfig); missingConfigErr {
 		// If no config file is used, the repo name must be set as a part of the envs.
+		log.Error(err.Error())
```

Comment:
In addition to just logging the error message returned, maybe we should add some context as a prefix to this log message, so that people understand it netter when seeing in the Frogbot log.

Reply:
done